### PR TITLE
Update footer style

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
In this PR I have updated footer styles and made it similar to the [teamdev.com](https://www.teamdev.com/). Also, I have refactored svg icons and move them to `img/icons` folder.

### CODE REVIEW

**Comment by JuliaEvseeva** [comment]
@andreySadomovskiy, PTAL.

File: _sass/modules/_footer.scss [hunk 1]
```diff
@@ -236,32 +218,70 @@ footer {
...
   }
 
   .icon-stack-overflow {
-    @include social-icon('../img/icon_stack-overflow.svg');
+    @include social-icon('../img/icons/icon-stack-overflow.svg');
   }
 
   .icon-youtube {
-    @include social-icon('../img/icon_youtube.svg');
+    @include social-icon('../img/icons/icon-youtube.svg');
```
Comment (andreySadomovskiy) [inline _sass/modules/_footer.scss]: These links `icon-stack-overflow `  and `icon-youtube` are commented. Do we need them in the future? If not remove HTML images and styles, please.

File: _sass/modules/_buttons.scss [hunk 2]
```diff
@@ -246,7 +246,7 @@
     margin-left: -20px;
     margin-top: -20px;
     content: "";
-    background: rgba($secondBrandColor, .26) url('../img/caret.svg') no-repeat;
+    background: rgba($secondBrandColor, .26) url('../img/icons/caret.svg') no-repeat;
     border-radius: 5px;
     width: 40px;
```
Comment (andreySadomovskiy) [inline _sass/modules/_buttons.scss]: I think we can extend icons variables and use them for icons sizes. 
```
$icon-size--xs              : 16px;
$icon-size--s               : 24px;
$icon-size--m               : 32px;
```

File: _sass/modules/_footer.scss (hunk 1 above)
Comment (JuliaEvseeva) [reply _sass/modules/_footer.scss]: These icons will be used in the future.

**Pull request merged**